rllib/algorithms/algorithm_config.py

```diff
@@ -1138,6 +1152,71 @@ class directly. Note that this arg can also be specified via
             logger_creator=self.logger_creator,
         )
 
+    def build_learner_group(
```

Comment:
New APIs: Learner and LearnerGroup can now be directly built from within a config object.